cwms-data-api/src/test/resources/cwms/cda/data/dto/project2/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "location" : {
+    "office-id" : "SPK",
+    "name" : "Project Id",
+    "timezone-name": "UTC"
+  },
+  "federal-cost" : 100.0,
+  "non-federal-cost" : 50.0,
+  "cost-year" : 1717199914902,
+  "cost-unit" : "$",
+  "federal-o-and-m-cost" : 10.0,
+  "non-federal-o-and-m-cost" : 5.0,
+  "authorizing-law" : "Authorizing Law",
+  "project-owner" : "Project Owner",
+  "hydropower-desc" : "Hydropower Description",
+  "sedimentation-desc" : "Sedimentation Description",
+  "downstream-urban-desc" : "Downstream Urban Description",
+  "bank-full-capacity-desc" : "Bank Full Capacity Description",
+  "pump-back-location" : {
```

Comment:
Same question with these locations as above

Reply:
The representation is able to hold full Locations in these properties